CHANGES.rst

```diff
@@ -5,7 +5,8 @@ Changelog
 0.12.1 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- Add transport header support for thrift call.
+- Fix zipkin trace.
```

Comment:
- Add support for specifying transport headers for Thrift clients.

BTW, if I understand correctly, the fix for Zipkin tracing for Thrift clients was basically to start passing the shard key, right? In that case, change the second one to,

    - Always pass ``shardKey``s for TCollector tracing calls. This fixes Zipkin tracing for Thrift clients.

Basically, saying "Fixed X" isn't very informative. It's as good as "It doesn't work" in bug reports.